lib/beaker/hypervisor/vagrant.rb

```diff
@@ -46,6 +46,20 @@ def make_vfile hosts, options = {}
           v_file << "    v.vm.synced_folder '.', '/vagrant', :nfs => true\n"
         end
 
+        if /freebsd/i.match(host['platform'])
+          v_file << "    v.ssh.shell = 'sh'\n"
+          v_file << "    v.vm.guest = :freebsd\n"
+
+          # FreeBSD NFS has a character restriction of 88 characters
+          # So you can enable it but if your module has a long name it probably won't work...
+          # Further links and reading in git commit message
+          if host['vagrant_freebsd_nfs'].nil?
```

Comment:
Hmm, honestly I prefer having notes in git commits, as git commits are more permanent and searchable than code, but I can do both :+1: